### PR TITLE
Proof IMU_COMPENSATION_PACKAGE #587

### LINKED ISSUE #587: Proof IMU_COMPENSATION_PACKAGE
Proof read transcribed code of [IMU_COMPENSATION_PACKAGE](//github.com/chrislgarry/Apollo-11/blob/master/Luminary099/IMU_COMPENSATION_PACKAGE.agc) against scans

Lines: 418
Page: 0326—0337 (11)

Reduced quality scans can be found [here][1]

[1]://ibiblio.org/apollo/ScansForConversion/Luminary099

### PR DESCRIPTION
**Pull request by ShadowMitia**
Closes #587.

### CODE REVIEW

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 1]
```diff
@@ -127,10 +127,10 @@ IRIGX		EXTEND
 		DCS	DELVY		# 	(PIPA PULSES) X 2(+14)
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:127]: 4 lines above, replace with:

```diff
-# (GYRO PULSES)/(PIPA PULSE) X 2(-6)		*
+# (GYRO PULSES)/(PIPA PULSE) X 2(-6)	 *
```

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:127]: 3 lines above, replace with:

```diff
-# -(ADIAX)(PIPAX)	   (GYRO PULSES) X 2(+14)
+# -(ADIAX)(PIPAX)   (GYRO PULSES) X 2(+14)

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:127]:
```suggestion
		DCS	DELVY		# (PIPA PULSES) X 2(+14)
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 2]
```diff
@@ -127,10 +127,10 @@ IRIGX		EXTEND
...
 		DXCH	MPAC		#
 		CS	ADSRAX		# 	(GYRO PULSES)/(PIPA PULSE) X 2(-6)	*
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:129]:
```suggestion
		CS	ADSRAX		# (GYRO PULSES)/(PIPA PULSE) X 2(-6)	 *
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 3]
```diff
@@ -127,10 +127,10 @@ IRIGX		EXTEND
...
-		TC	GCOMPSUB	# 	-(ADSRAX)(PIPAY)   (GYRO PULSES) X 2(+14)
+		TC	GCOMPSUB	# 	+(ADSRAX)(PIPAY)   (GYRO PULSES) X 2(+14)
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:130]:
```suggestion
		TC	GCOMPSUB	# +(ADSRAX)(PIPAY)  (GYRO PULSES) X 2(+14)
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 4]
```diff
@@ -127,10 +127,10 @@ IRIGX		EXTEND
...
 
 #		EXTEND			# ***
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:132]:
```suggestion
#		EXTEND		  # ***
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 5]
```diff
@@ -127,10 +127,10 @@ IRIGX		EXTEND
...
-#		DCS	DELVY		# ***	(PIPA PULSES) X 2(+14)
+#		DCS	DELVZ		# ***	(PIPA PULSES) X 2(+14)
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:133]:
```suggestion
#		DCS	DELVZ	  # ***   (PIPA PULSES) X 2(+14)
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 6]
```diff
@@ -127,10 +127,10 @@ IRIGX		EXTEND
...
 #		DXCH	MPAC		# ***
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:134]:
```suggestion
#		DXCH	MPAC	  # ***
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 7]
```diff
@@ -127,10 +127,10 @@ IRIGX		EXTEND
...
 #		CA	ADOAX		# ***	(GYRO PULSES)/(PIPA PULSE) X 2(-6)	*
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:135]:
```suggestion
#		CA	ADOAX	  # ***   (GYRO PULSES)/(PIPA PULSE) X 2(-6)	 *
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 8]
```diff
@@ -127,10 +127,10 @@ IRIGX		EXTEND
...
 #		TC	GCOMPSUB	# ***	-(ADOAX)(PIPAZ)	   (GYRO PULSES) X 2(+14)
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:136]:
```suggestion
#		TC	GCOMPSUB  # ***   -(ADOAX)(PIPAZ)   (GYRO PULSES) X 2(+14)
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 9]
```diff
@@ -155,7 +155,7 @@ IRIGY		EXTEND
 #		DCS	DELVX		# ***	(PIPA PULSES) X 2(+14)
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:155]: Line 145 (above), replace with:

```diff
-# (GYRO PULSES)/(PIPA PULSE) X 2(-6)		*
+# (GYRO PULSES)/(PIPA PULSE) X 2(-6)	 *

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:155]: Line 146 (above), replace with:

```diff
-# -(ADIAY)(PIPAY)	   (GYRO PULSES) X 2(+14)
+# -(ADIAY)(PIPAY)   (GYRO PULSES) X 2(+14)

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:155]: Line 151 (above), replace with:

```diff
-# (GYRO PULSES)/(PIPA PULSE) X 2(-6)		*
+# (GYRO PULSES)/(PIPA PULSE) X 2(-6)	 *

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:155]: Line 152 (above), replace with:

```diff
-# +(ADSRAY)(PIPAZ)      	(GYRO PULSES) X 2(+14)
+# +(ADSRAY)(PIPAZ)  (GYRO PULSES) X 2(+14)

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:155]: 1 line above, replace with:

```diff
-#		EXTEND			# ***
+#		EXTEND		  # ***

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:155]:
```suggestion
#		DCS	DELVX	  # ***   (PIPA PULSES) X 2(+14)
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 10]
```diff
@@ -155,7 +155,7 @@ IRIGY		EXTEND
...
 #		DXCH	MPAC		# ***
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:156]:
```suggestion
#		DXCH	MPAC	  # ***
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 11]
```diff
@@ -155,7 +155,7 @@ IRIGY		EXTEND
...
 #		CA	ADOAY		# ***	(GYRO PULSES)/(PIPA PULSE) X 2(-6)	*
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:157]:
```suggestion
#		CA	ADOAY	  # ***   (GYRO PULSES)/(PIPA PULSE) X 2(-6)	 *
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 12]
```diff
@@ -155,7 +155,7 @@ IRIGY		EXTEND
...
-#		TC	GCOMPSUB	# ***	-(ADOAY)(/PIPAX)   (GYRO PULSES) X 2(+14)
+#		TC	GCOMPSUB	# ***	-(ADOAY)(PIPAX)   (GYRO PULSES) X 2(+14)
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:158]:
```suggestion
#		TC	GCOMPSUB  # ***   -(ADOAY)(PIPAX)   (GYRO PULSES) X 2(+14)
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 13]
```diff
@@ -155,7 +155,7 @@ IRIGY		EXTEND
...
 
 		TC	MPAC +2
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:161]: Line 167 (below), replace with:

```diff
-# (GYRO PULSES)/(PIPA PULSE) X 2(-6)		*
+# (GYRO PULSES)/(PIPA PULSE) X 2(-6)	 *

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:161]: Line 169 (below), replace with:

```diff
-# -(ADSRAZ)(PIPAY)	  (GYRO PULSES) X 2(+14)
+# -(ADSRAZ)(PIPAY)  (GYRO PULSES) X 2(+14)

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:161]: Line 174 (below), replace with:

```diff
-# (GYRO PULSES)/(PIPA PULSE) X 2(-6)		*
+# (GYRO PULSES)/(PIPA PULSE) X 2(-6)	 *

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:161]: Line 175 (below), replace with:

```diff
-# -(ADIAZ)(PIPAZ)	   (GYRO PULSES) X 2(+14)
+# -(ADIAZ)(PIPAZ)   (GYRO PULSES) X 2(+14)

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:161]: Lines 177-181 (below), replace with:

```diff
-#		EXTEND			# ***
-#		DCS	DELVX		# ***	(PIPA PULSE) X 2(+14)
-#		DXCH	MPAC		# ***
-#		CS	ADOAZ		# ***	(GYRO PULSES)/(PIPA PULSE) X 2(-6)	*
-#		TC	GCOMPSUB	# ***	+(ADOAZ)(PIPAX)	   (GYRO PULSES) X 2(+14)
+#		EXTEND		  # ***   
+#		DCS	DELVX	  # ***   (PIPA PULSE) X 2(+14)
+#		DXCH	MPAC	  # ***   
+#		CS	ADOAZ	  # ***   (GYRO PULSES)/(PIPA PULSE) X 2(-6)	 *
+#		TC	GCOMPSUB  # ***   +(ADOAZ)(PIPAX)   (GYRO PULSES) X 2(+14)

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 14]
```diff
@@ -216,14 +216,14 @@ DRIFTSUB	EXTEND
 		QXCH	BUF 	+1
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:216]: Line 188 (above), replace with:

```diff
-# (GYRO PULSES)/(PIPA PULSE) X 2(-6)		*
+# (GYRO PULSES)/(PIPA PULSE) X 2(-6)	 *
```

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:216]: Line 189 (above), replace with:

```diff
-# NOW = (GYRO PULSES) X 2(+8)			*
+# NOW = (GYRO PULSES) X 2(+8)		 *

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:216]: Line 196 (above), replace with:

```diff
-# NOW = (GYRO PULSES) X 2(+8)			*
+# NOW = (GYRO PULSES) X 2(+8)		 *

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:216]: Line 198 (above), `-` instead of `--`

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:216]: Line 200 (above), replace with:

```diff
-# SCALE 2(+6)	SHIFT RIGHT 6			*
+# SCALE 2(+6)	SHIFT RIGHT 6		 *

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:216]: Line 202 (above), replace with:

```diff
-# HI(ADIA)(PIPAI) OR HI(ADSRA)(PIPAI)
+# HI(ADIA)(PIPAI)  OR  HI(ADSRA)(PIPAI)

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:216]: Line 204 (above), `-` instead of `--`

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:216]: Line 206 (above), replace with:

```diff
-# SCALE 2(+6)	SHIFT RIGHT 6			*
+# SCALE 2(+6)	SHIFT RIGHT 6		 *

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:216]: Line 210 (above), replace with:

```diff
-# (ADIA)(PIPAI) OR (ADSRA)(PIPAI)
+# (ADIA)(PIPAI)  OR  (ADSRA)(PIPAI)

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 15]
```diff
@@ -216,14 +216,14 @@ DRIFTSUB	EXTEND
...
 
 		EXTEND			# C(A) = NBD	(GYRO PULSES)/(CS) X 2(-5)
-		MP	1/PIPADT	# (CS) X 2(+8)	NO (GYRO PULSES) X 2(+3)
+		MP	1/PIPADT	# (CS) X 2(+8)	NOW (GYRO PULSES) X 2(+3)
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:219]:
```suggestion
		MP	1/PIPADT	# (CS) X 2(+8)	 NOW (GYRO PULSES) X 2(+3)
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 16]
```diff
@@ -216,14 +216,14 @@ DRIFTSUB	EXTEND
...
 		LXCH	MPAC 	+1	# SAVE FOR FRACTIONAL COMPENSATION
 		EXTEND
 		MP	BIT4		# SCALE 2(+11)	SHIFT RIGHT 11
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:222]:
```suggestion
		MP	BIT4		# SCALE 2(+11)	   SHIFT RIGHT 11
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 17]
```diff
@@ -216,14 +216,14 @@ DRIFTSUB	EXTEND
...
 		INDEX	BUF
 		DAS	GCOMP		# HI(NBD)(DELTAT)	(GYRO PULSES) X 2(+14)
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:224]:
```suggestion
		DAS	GCOMP		# HI(NBD)(DELTAT)   (GYRO PULSES) X 2(+14)
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 18]
```diff
@@ -216,14 +216,14 @@ DRIFTSUB	EXTEND
...
 
-		CA	MPAC 	+1	# NO MINOR PART
+		CA	MPAC 	+1	# NOW MINOR PART
 		EXTEND
 		MP	BIT4		# SCALE 2(+11)		SHIFT RIGHT 11
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:228]:
```suggestion
		MP	BIT4		# SCALE 2(+11)	   SHIFT RIGHT 11
```

File: Luminary099/IMU_COMPENSATION_PACKAGE.agc [hunk 19]
```diff
@@ -216,14 +216,14 @@ DRIFTSUB	EXTEND
...
 		TS	L
```
Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:229]: Line 232 (below), replace with:

```diff
-# (NBD)(DELTAT)		(GYRO PULSES) X 2(+14)
+# (NBD)(DELTAT)	    (GYRO PULSES) X 2(+14)

Comment (wopian) [inline Luminary099/IMU_COMPENSATION_PACKAGE.agc:229]: Line 244 (below), replace `--` with `-`

**Review by wopian — CHANGES_REQUESTED** [review]
Review up to page 332. Pages 333 to 337 have not been reviewed yet.

I've checked and committed several suggestions for you. Some requested changes were outside of range that you'll need to commit those yourself manually.

This file really lends itself to using rulers in a code editor as the file is filled with characters that hug the right-most column of the page (column 81 - technically 80 without the `#`). If you're using Visual Studio Code, this is a good ruler to use in `settings.json` and can be achieved in all other editors:

```json
"editor.rulers": [8, 16, 24, 32, 40, 42, 81, 82]
```

**Comment by ShadowMitia** [comment]
Sorry about the poor commit, I guess I didn't read how to contribute well enough...

I should've applied all of your changes, and I redid a proofread and should've aligned everything correctly now.

**Review by wopian — APPROVED** [review]

**Comment by wopian** [comment]
> Sorry about the poor commit, I guess I didn't read how to contribute well enough...

No worries, you did well for a first-time proof read with spotting most of the typos

**Comment by ShadowMitia** [comment]
Thanks!

**Pull request merged**